yarn-project/prover-client/src/prover-agent/memory-proving-queue.ts

```diff
@@ -427,7 +425,7 @@ export class MemoryProvingQueue implements ServerCircuitProver, ProvingJobSource
     inputs: AvmCircuitInputs,
     signal?: AbortSignal,
     epochNumber?: number,
-  ): Promise<ProofAndVerificationKey<Proof>> {
+  ): Promise<ProofAndVerificationKey<number>> {
```

Comment:
Wonder if type Unbounded = number is too hacky :)

Reply:
Ah good shout! I think it would it clean and I can leave a comment at the definition point about it